src/ontology/cl-odk.yaml

```diff
@@ -100,6 +100,8 @@ import_group:
       mirror_from: http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim-disjoint-over-in-taxon.owl
     - id: omo
       module_type: mirror
+    - id: mbao
+      mirror_from: https://raw.githubusercontent.com/brain-bican/mouse_brain_atlas_ontology/main/mbao-base.owl
```

Comment:
Since the BICAN maintains its own PURLs, why not use them? Why not downloading from `https://purl.brain-bican.org/ontology/mbao/mbao-base.owl`?

Reply:
BICAN PURL server is performing poorly now. While the issue isn't noticeable in the browser, approximately half of the wget operations are failing. Since the server is managed by the Allen Institute, it's not something we can resolve immediately on our end. As a workaround, we've opted for a more stable approach for now.